edelphi/src/main/webapp/_scripts/gui/panel/queryblockcontroller.js

```diff
@@ -189,14 +178,14 @@ var QueryBlockController = Class
           },
 
           disableNext : function() {
+            this._disableNextButton();
+            
             var button = this._nextButton || this._finishButton;
 
             var originalTitle = $(button).readAttribute('data-original-title');
             var title = $(button).readAttribute('title');
 
-            $(button).writeAttribute('disabled', 'disabled');
-            $(button).writeAttribute('title',
-                getLocale().getText('query.buttonDisabledNotReplied'));
+            $(button).writeAttribute('title', getLocale().getText('query.buttonDisabledNotReplied'));
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') "getLocale" does not exist. Change its name or declare it so that its usage doesn't result in a "ReferenceError". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS3827)